tests/integration/targets/aci_l3out_bgp_peer/tasks/main.yml

```diff
@@ -201,6 +201,7 @@
       address_type_controls:
         - af-ucast
       ttl: 2
+      local_as_number_config: "replace-as"
```

Comment:
What happens when you want to do a change of config? Did you add tests for changing the local_as_number_config input on an existing l3out bgp peer?

Reply:
I just pushed changes to check the update operation of local_as_number_config.